index.css

```diff
@@ -96,3 +102,78 @@ main {
     margin: auto;
     text-align: center;
 }
+
+
+.wrapper {
+    margin: 0 auto;
+    background-color: #ffffff;
+    height: 50px;
+    padding: 0;
+
+
+
+}
+
+
+header {
+    display: flex;
+    justify-content: space-between;
+    padding: 5px;
+    border-radius: 0;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    background-color: #ffffff;
+
+
+}
+
+header img.logo {
+    padding: 5px;
+    width: 40px;
+    height: 40px;
+}
+
+header img.profile {
+    width: 40px;
+    height: 40px;
+    padding: 5px;
+    border-radius: 50%;
+
+
+}
+
+.search-container{
+    border: 2px;
+
+
+}
+.search-container input[type=text] {
+    padding: 2px;
+    margin-top: 5px;
+    font-size: 20px;
+    width: 400px;
+    height: 40px;
+    border: thin solid gainsboro;;
+
+
+}
```

Comment:
I would suggest changing the design and attributes as following to be more neat and automatic. This way, search box also takes automatically 80% of the container width.

.search-container input[type=text] {
    width: 80%;
    height: 80px;
    padding: 20px;
    font-size: 28px;
    border: thin solid gainsboro;
    background-color: rgb(245, 240, 240);
}